dashboard/app/models/level.rb

```diff
@@ -186,6 +186,20 @@ def self.key_to_params(key)
     end
   end
 
+  # Returns true if this is a pixelation level.
+  def pixelation?
+    # TODO(dave|joshlory): Define Pixelation < DSLDefined as a new level type,
+    # eliminating this fragile test of 'href'.
+    self.is_a?(DSLDefined) && self.try(:properties).try(:[], "href") == "pixelation/pixelation.html"
```

Comment:
I think because of how `serialized_attrs` are implemented there will always be a `properties` hash for all level types:
```
irb(main):001:0> Level.all.all?(&:properties)
  Level Load (14.8ms)  SELECT `levels`.* FROM `levels`
=> true
```
@wjordan would know for sure.